docs/src/main/asciidoc/openapi-swaggerui.adoc

```diff
@@ -385,6 +406,36 @@ Alternative paths are:
 Live reload of static OpenAPI document is supported during development. A modification to your OpenAPI document will be picked up on fly by Quarkus.
 ====
 
+== Changing the openapi version
+
+By default, when the document is generated, the openapi version used will be `3.0.3`. If you use a static file as mentioned above, the version in the file 
+will be used. You can also define the version in SmallRye using the following configuration:
+
+[source, properties]
+----
+mp.openapi.extensions.smallrye.openapi=3.0.2
+----
+
+This might be useful if your API go through a Gateway that needs a certain version.
+
+== Auto-generation of OperationId
+
+The https://swagger.io/docs/specification/paths-and-operations/[Operation Id] can be set using the `@Operation` annotation, and is in many cases useful when using a tool to generate a client stub from the schema. 
+The Operation Id is typically used for the method name in the client stub. In SmallRye, you can auto-generate this Operation Id by using the following configuration:
+
+[source, properties]
+----
+mp.openapi.extensions.smallrye.operationIdStrategy=METHOD
+----
+
+Now you do not need to use the `@Operation` annotation. While generating the document, the method name will be used for the Operation Id.
+
+The strategies available for generating the Operation Id is:
```

Comment:
```suggestion
The strategies available for generating the Operation Id are:
```

Reply:
Also wondering if this should be a table, just like [Database kind to JDBC driver mapping](https://github.com/quarkusio/quarkus/blob/df5ee4e1974b280eba81b633c48b0a7d08c37f72/docs/src/main/asciidoc/datasource.adoc#configure-the-jdbc-connection) in the `datasource.adoc` guide

Reply:
Agree. Made it a table